types/lodash/index.d.ts

```diff
@@ -9859,32 +9859,32 @@ declare namespace _ {
          * object === other;
          * // => false
          */
-        isEqual(
-            value: any,
-            other: any
+        isEqual<T1, T2>(
```

Comment:
This change doesn't really make sense. If this function accepts a `T1` and a `T2` for arbitrary values of `T1` and `T2`, that's the same as saying it accepts `any`.